fastchat/serve/gradio_block_arena_vision_anony.py

```diff
@@ -483,11 +578,12 @@ def build_side_by_side_vision_ui_anony(context: Context, random_questions=None):
         )
 
     with gr.Row() as button_row:
-        if random_questions:
-            global vqa_samples
-            with open(random_questions, "r") as f:
-                vqa_samples = json.load(f)
-            random_btn = gr.Button(value="🔮 Random Image", interactive=True)
+        random_btn = gr.Button(value="🔮 Random Image", interactive=True) 
+        # if random_questions:
+        #     global vqa_samples
+        #     with open(random_questions, "r") as f:
+        #         vqa_samples = json.load(f)
+        #     random_btn = gr.Button(value="🔮 Random Image", interactive=True)
```

Comment:
we shouldn't remove these..

Reply:
I'm not sure how random_question works but the current implementation is broken when the if condition isn't true because I get an error that random_btn has no value.